docs/1000.yaml

```diff
@@ -80,6 +80,10 @@ Q179208:
 
 Q179467:
   title: Fourier theorem
+  decls:
+    # further generalisations possible; the wikipedia page is *really* vague
```

Comment:
There is an official `comment` field, right? Maybe worth writing this there to encourage further work?

Also what about including `year` and `authors`? It seems a lot of declarations have been added to this file without those fields but maybe it's worth thinking about whether we should encourage them to be added in the future.

bors d+